### PR TITLE
some doc changes

### PR DESCRIPTION
**Pull request by KristofferC**
Some word changes after a read through.

### CODE REVIEW

File: docs/src/man/binary_operators.md [hunk 1]
```diff
@@ -27,7 +27,7 @@ dot
 
 ## Double contraction
 
-Double contractions contracts the two most inner "legs" of the tensors. The result of a double contraction between a tensor of order `n` and a tensor of order `m` gives a tensor of order `m + n - 4`. For example, double contraction between two second order tensors ``\mathbf{B}`` and ``\mathbf{C}`` can be written as:
+A double contractions between two tensors contracts the two most inner indices. The result of a double contraction between a tensor of order `n` and a tensor of order `m` is a tensor of order `m + n - 4`. For example, double contraction between two second order tensors ``\mathbf{B}`` and ``\mathbf{C}`` can be written as:
```
Comment (fredrikekre) [inline docs/src/man/binary_operators.md]: contraction

File: docs/src/man/binary_operators.md [hunk 2]
```diff
@@ -41,7 +41,7 @@ dcontract
 
 ## Tensor product (open product)
 
-Tensor products or open product of a tensor of order `n` and a tensor of order `m` gives a tensor of order `m + n`.  For example, open product between two vectors ``\mathbf{b}`` and ``\mathbf{c}`` can be written as:
+The tensor products (or open product) between a tensor of order `n` and a tensor of order `m` is a tensor of order `m + n`.  For example, open product between two vectors ``\mathbf{b}`` and ``\mathbf{c}`` can be written as:
```
Comment (fredrikekre) [inline docs/src/man/binary_operators.md]: product

File: docs/src/man/constructing_tensors.md [hunk 3]
```diff
@@ -38,28 +41,55 @@ julia> zero(SymmetricTensor{4, 2})
 [:, :, 2, 2] =
  0.0  0.0
  0.0  0.0
+```
+
+
+## Random tensors
+
+A tensor with random zeros is created using the function `rand`, applied to the type of tensor that should be created:
```
Comment (fredrikekre) [inline docs/src/man/constructing_tensors.md]: random values

File: docs/src/man/constructing_tensors.md [hunk 4]
```diff
@@ -102,7 +133,11 @@ julia> tensor_data = reinterpret(Vec{2, Float64}, data, (5,))
  [0.794026,0.854147]
  [0.200586,0.298614]
  [0.246837,0.579672]
+```
 
+The data can also be reinterpreted back to a Julia `array`
```
Comment (fredrikekre) [inline docs/src/man/constructing_tensors.md]: `Array`

File: docs/src/man/constructing_tensors.md (hunk 3 above)
Comment (KristofferC) [reply docs/src/man/constructing_tensors.md]: random zeros LAL

Comment (fredrikekre) [reply docs/src/man/constructing_tensors.md]: `Float64` or `Float32`. Kappa

File: docs/src/man/constructing_tensors.md [hunk 5]
```diff
@@ -102,7 +133,11 @@ julia> tensor_data = reinterpret(Vec{2, Float64}, data, (5,))
...
+
+```
```
Comment (fredrikekre) [inline docs/src/man/constructing_tensors.md]:
``` julia
```

Comment (fredrikekre) [reply docs/src/man/constructing_tensors.md]: Nice comment. Meant this
```julia

Comment (fredrikekre) [reply docs/src/man/constructing_tensors.md]: or maybe jlcon to match the doctests.

Comment (KristofferC) [reply docs/src/man/constructing_tensors.md]: You mean doctest? Hmm, annoyting that I can't keep the variable  from above.

File: docs/src/man/constructing_tensors.md (hunk 3 above)
Comment (fredrikekre) [reply docs/src/man/constructing_tensors.md]: Keeping this? :)

Comment (KristofferC) [reply docs/src/man/constructing_tensors.md]: mi scusi

Comment (KristofferC) [reply docs/src/man/constructing_tensors.md]: updated

**Review by fredrikekre — APPROVED** [review]

**Pull request merged**